docs/api.md

```diff
@@ -1,3 +1,10 @@
+<p align="right">
```

Comment:
looks like these image directives will all need to be written in restructured text, or they won't render correctly... for example:

```
.. image:: images/cropped-plus3it-logo-cmyk.png
   :width: 140px
   :alt: Powered by Plus3 IT Systems
   :align: right
   :target: https://plus3it.com
```

http://docutils.sourceforge.net/docs/ref/rst/directives.html#image

Reply:
I've updated the image code to use RST.  The pages rendered correctly after generating them using sphinx on my local system.  Thanks for the reference